devDocs/technicalDesignOverview.md

```diff
@@ -67,6 +67,14 @@ Aside from accessibility and native APIs, Windows provides many functions which
 Information that can be obtained includes the class name of a window, the current foreground window and system battery status.
 Tasks that can be performed include moving/clicking the mouse and sending key presses.
 
+### Logging
+
+#### Logging in secure mode
+`logHandler.initialize` prevents logging in secure mode.
+This is because it is a security concern to log during secure mode (e.g. passwords are logged).
+To change this for testing, patch the source build.
+`nvda.log` files are then generated in the System profile's `%TEMP%` directory.
```

Comment:
I don't think we need to be this specific in the technicalDesignOverview.
Logging should be disabled in secure mode, not just secure screens.

It would be fine to amend this in a future release, such as:

```suggestion
When logging from a secure screen, `nvda.log` files are generated in the System profile's `%TEMP%` directory.
```

This paragraph in the technicalDesignOverview.md aims to answer:
* is logging disabled in secure mode? why?
* how can I change this for testing? This can be updated, but the current description works too.
* Where can I find these logs? It is difficult to guess this when logging from a secure screen.